dynamic_rest/fields/fields.py

```diff
@@ -49,17 +51,36 @@ def to_representation(self, value):
     def to_internal_value(self, data):
         return data
 
+    def get_meta_arg(self, local_kwarg, meta_attr):
+        """ Get arguments that can either be set on the field itself,
+        or on a Meta attribute (e.g. "read_only").
+        """
+
+        # Explicit kwarg on field takes precedence
+        if local_kwarg in self.kwargs:
+            return self.kwargs[local_kwarg]
+
+        # Otherwise infer from parent's Meta attr
+        parent_meta_attr = getattr(self.parent.Meta, meta_attr, [])
+        return self.field_name in parent_meta_attr
+
     def bind(self, *args, **kwargs):
         super(DynamicField, self).bind(*args, **kwargs)
 
-        # Infer read_only from parent Meta.read_only_fields
-        if 'read_only' not in self.kwargs:
-            parent_ro_fields = getattr(
-                self.parent.Meta,
-                'read_only_fields',
-                []
-            )
-            self.read_only = self.field_name in parent_ro_fields
+        self.read_only = self.get_meta_arg('read_only', 'read_only_fields')
+        self.immutable = self.immutable or self.get_meta_arg(
```

Comment:
Nevermind, I see that `self.kwargs` is something that we're doing to capture all DRF-only kwargs